ADAL/ADALAutomation/ADAutoRequestViewController.m

```diff
@@ -21,11 +21,46 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#import <UIKit/UIKit.h>
+#import "ADAutoRequestViewController.h"
+#import "ADLogger+Internal.h"
 
-@interface ADAutoTextAndButtonView : UIView
+@interface ADAutoRequestViewController ()
 
-@property (readonly) UIButton* actionButton;
-@property (readonly) UITextView* textView;
+@property (strong, nonatomic) IBOutlet UITextView *requestInfo;
+@property (strong, nonatomic) IBOutlet UIButton *requestGo;
```

Comment:
Agreed, but the accessibilityIdentifier of this button was asked to be requestGo, and I wanted the name to be in sync.